README.md

```diff
@@ -16,19 +16,17 @@
 
 <br>
 
-ReactPHP is a low-level library for event-driven programming in PHP. At its core
-is an event loop, on top of which it provides low-level utilities, such as:
+ReactPHP is a low-level library for event-driven programming in PHP. The core of ReactPHP is an event loop built on top of the low-level utilities, such as:
 Streams abstraction, async DNS resolver, network client/server, HTTP
 client/server and interaction with processes. Third-party libraries can use these
 components to create async network clients/servers and more.
 
 ReactPHP is production ready and battle-tested with millions of installations
-from all kinds of projects around the world. Its event-driven architecture makes
-it a perfect fit for efficient network servers and clients handling hundreds or
+from all kinds of projects around the world. The event-driven architecture of ReactPHP creates efficient network servers and clients handling hundreds or
```

Comment:
This suggests that "the architecture […] creates servers", while this meant to say "the architecture is a good fit".